### PR TITLE
https://jira.baeldung.com/browse/BAEL-7696

### PR DESCRIPTION
**Pull request by Michaelin007**

### CODE REVIEW

File: spring-reactive-modules/spring-reactive-client-2/src/main/java/com/baeldung/fluxmapandondonext/DoOnNext.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.baeldung.fluxmapandondonext;
+
+import reactor.core.publisher.Flux;
+
+public class DoOnNext {
+
+    public Flux<Integer> number_service_range() {
+        return Flux.just(10, 11, 12, 13, 14);
+    }
+}
```
Comment (ashleyfrieze) [inline spring-reactive-modules/spring-reactive-client-2/src/main/java/com/baeldung/fluxmapandondonext/DoOnNext.java:10]: Do we need this and `FluxMap.java`?

File: spring-reactive-modules/spring-reactive-client-2/src/test/java/com/baeldung/fluxmapandondonext/DoOnNextUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                LOGGER.info(String.valueOf(number));
+                counter.incrementAndGet();
+            })
+            .onErrorResume(Flux::error);
+
+        StepVerifier.create(numberFlux)
+            .expectNextCount(5)
+            .verifyComplete();
+
+        Assertions.assertEquals(counter.get(), 5);
```
Comment (ashleyfrieze) [inline spring-reactive-modules/spring-reactive-client-2/src/test/java/com/baeldung/fluxmapandondonext/DoOnNextUnitTest.java:31]: Let's `static import` the `assertEquals` function here. Also, note that the expected is the first parameter in JUnit assertions, not the second as you have it here.

File: spring-reactive-modules/spring-reactive-client-2/src/main/java/com/baeldung/fluxmapandondonext/DoOnNext.java (hunk 1 above)
Comment (Michaelin007) [reply spring-reactive-modules/spring-reactive-client-2/src/main/java/com/baeldung/fluxmapandondonext/DoOnNext.java]: Hi @ashleyfrieze, both DoOnNext.java and FluxMap.java removed.

File: spring-reactive-modules/spring-reactive-client-2/src/test/java/com/baeldung/fluxmapandondonext/DoOnNextUnitTest.java (hunk 2 above)
Comment (Michaelin007) [reply spring-reactive-modules/spring-reactive-client-2/src/test/java/com/baeldung/fluxmapandondonext/DoOnNextUnitTest.java]: Done.
Thanks a lot.

**Pull request merged**